package.json

```diff
@@ -98,9 +105,9 @@
     "react-blockies": "^1.4.0",
     "react-display-name": "^0.2.3",
     "react-onclickout": "^2.0.8",
-    "react-spring": "^7.2.11",
+    "react-spring": "7.2.10",
     "recursive-copy": "^2.0.9",
-    "use-inside": "^0.2.0"
+    "use-inside": "https://github.com/aragon/use-inside#16f321e499d8f89e9b0fb0999e2d21db2b119bbf"
```

Comment:
On bundle api-extractor cannot found definition of use-inside on the last version published on npm, but on currenty master (referenced by commit for safety install), have the TS definitions updated